pkg/hosts/hosts.go

```diff
@@ -29,7 +29,7 @@ type Hosts struct {
 }
 
 func init() {
-	// goodhosts unconditionnally uses this environment variable
+	// goodhosts unconditionally uses this environment variable
```

Comment:
eh, I always get it wrong, and thought during typing that I should check where the double consonants are ;) Thanks!